### PR TITLE
Editorial and spelling fixes; fix to hit testing

### PR DESCRIPTION
**Pull request by chrishtr**
I also added a paragraph at the beginning of the chapter putting the concerns of permanently-disabled people
first. This is in response to feedback from an a11y expert review who suggested that change.

### CODE REVIEW

File: src/lab14.py [hunk 1]
```diff
@@ -668,9 +668,10 @@ def intersects(self, x, y):
         return False
 
     def hit_test(self, x, y):
-        for node in tree_to_list(self, []):
-            if node.bounds.intersects(x, y):
-                return node
+        nodes = [node for node in tree_to_list(self, [])
```
Comment (chrishtr) [inline src/lab14.py:671]: The old code was wrong because you need to hit test from the bottom of the tree up.

File: src/lab14.py [hunk 2]
```diff
@@ -1742,7 +1739,8 @@ def handle_click(self, e):
         self.lock.release()
 
     def handle_hover(self, event):
-        if not self.accessibility_is_on:
+        if not self.accessibility_is_on or \
```
Comment (chrishtr) [inline src/lab14.py:1742]: This fixes a race condition that happened sometimes when enabling a11y.

File: book/accessibility.md [hunk 3]
```diff
@@ -106,7 +115,7 @@ In the United States, the European Union, and many other countries,
 website accessibility is legally required. For example, United States
 Government websites are required to be accessible under [Section
 508][sec508] of the [Rehabilitation Act Amendments of
-1993][rehab-act], and the government has a bunch of
+1973][rehab-act], and the government has a bunch of
```
Comment (pavpanchekha) [inline book/accessibility.md:118]: I think, confusingly, these are two similarly-named acts. We should double-triple check.

File: book/accessibility.md [hunk 4]
```diff
@@ -883,7 +894,7 @@ class Tab:
 
 Finally, we just need to focus on the chosen element. If we've reached
 the last the focusable node (or if there weren't any focusable nodes
-to begin with), we'll unfocus the page and move focus to the address
+to begin with), we'll UN-focus the page and move focus to the address
```
Comment (pavpanchekha) [inline book/accessibility.md:897]: Would prefer italics rather than capitalization for emphasis here.

File: src/lab14.py (hunk 1 above)
Comment (pavpanchekha) [reply src/lab14.py:671]: Oh yeah, you are right.

**Review by pavpanchekha — APPROVED** [review]
Please consider at least re-checking which Rehabilitation Act we want. I think a semi-authoritative source is this: https://www.section508.gov/manage/laws-and-policies/

Which makes me think it is 1998, not 1993 but also not 1973, that we need to cite.

**Comment by pavpanchekha** [comment]
And yes, the alert examples works for me in VoiceOver+Firefox

**Comment by pavpanchekha** [comment]
But it doesn't work in VoiceOver+Safari.

**Comment by pavpanchekha** [comment]
I guess either way we should explain that this example will only work in some browsers and screen readers.

**Comment by pavpanchekha** [comment]
Well, to finish data collection, the example works on Chrome and Firefox, but not Safari, with VoiceOver.

File: book/accessibility.md (hunk 3 above)
Comment (chrishtr) [reply book/accessibility.md:118]: I looked at the site you recommended, and it's the ...act of 1973, with amendments made in 1998. I added a parenthetical.

File: book/accessibility.md (hunk 4 above)
Comment (chrishtr) [reply book/accessibility.md:897]: That was a typo - changed to "un-focus"

**Pull request merged**